src/pet.js

```diff
@@ -17,6 +17,7 @@ function Pet(name) {
    this.age = INITIAL_AGE;
    this.hunger = INITIAL_HUNGER;
    this.fitness = INITIAL_FITNESS;
+   this.children = [];
 }
 
 Pet.prototype = {
```

Comment:
To improve the readability of your code, I would suggest to have all the methods inside of this `Pet.prototype` object. In that way, each method would have the syntax: `methodName() {}` , instead of having to write Pet.prototype countless times.